### ISSUE TITLE
References incorrect respository

### ISSUE THREAD

**Issue by RattyDAVE** [open-simh/simh#258]
https://github.com/open-simh/simh/blob/d901d676c452728567c7d123d875e6ddb5d47fef/makefile#L1095

**Comment by pkoning2** [comment]
True, but it's a bit more complicated than that.  There are also links to things under "windows-build" which haven't been carried over to open-simh.

**Comment by al20878** [comment]
> it's a bit more complicated than that

While it may be so, what's preventing to change `makefile` to reference the `open-simh` repository correctly already?
```
$ grep -n simh/simh makefile
1095:        $(info *** Error ***      $$ git clone https://github.com/simh/simh)
1100:        $(info *** Error ***      $$ wget(or via browser) https://github.com/simh/simh/archive/master.zip)
```
Both can be switched to print `https://github.com/open-simh/simh`

**Comment by al20878** [comment]
@RattyDAVE  :  Makefile should reference the correct repo now, please check
